docs/strategy-advanced.md

```diff
@@ -179,6 +236,55 @@ class AwesomeStrategy(IStrategy):
             return (-0.07 + current_profit)
         return 1
 ```
+#### Custom stoploss using an indicator from dataframe example
+
+Imagine you want to use `custom_stoploss()` to use a trailing indicator like e.g. "ATR"
+
+See: "Storing custom information using DatetimeIndex from `dataframe`" example above) on how to store the indicator into `custom_info`
+
+!!! Warning
+    only use .iat[-1] in live mode, not in backtesting/hyperopt
+    otherwise you will look into the future
+    see: https://www.freqtrade.io/en/latest/strategy-customization/#common-mistakes-when-developing-strategies
```

Comment:
```suggestion
    see [Common mistakes when developing strategies](strategy-customization.md#common-mistakes-when-developing-strategies) for more info.
```

we'll want to use relative links - so it'll also work in the future, when looking at a "fixed" version of the docs.